book/installation.rst

```diff
@@ -269,6 +269,14 @@ If there are any issues, correct them now before moving on.
 
         setfacl isn't available on NFS mount points. However, setting cache
         and logs over NFS is strongly not recommended for performance.
+        
+    .. note::
+
+        when running php-fpm you should make sure that your user matches 
+        webserver one or add additional rules using setfacl for user running php.
```

Comment:
Can we reword this a bit? I think something like "When not using PHP's built-in web server, make sure that the web server process runs with your user account or to configure advanced rules (like ACLs)."